mods/seasonal/moves.js

```diff
@@ -1948,7 +2005,7 @@ let BattleMovedex = {
 		onHit: function (target, source) {
 			this.useMove("healorder", source, source);
 			this.useMove("defendorder", source, source);
-			this.useMove("attackorder", source, source);
+			this.useMove("attackorder", source); // Move fails otherwise
```

Comment:
My bad, this is an attack move from a self targeting move so it needs to be `source, source.side.foe.active[0]`